### PR TITLE
Upgrade to arcgis earth v1.11

### PR DESCRIPTION
**Pull request by yong7743**
Will merge it after we released 1.11.

- [x] Code review @bentan2013
- [x] Code review @gis2all 
- [x] Code review @xuew8412

### CODE REVIEW

**Review by xuew8412 — APPROVED** [review]
LGTM. I also tested local and it worked well.

**Review by bentan2013 — APPROVED** [review]
looks good for me

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ ArcGIS Earth Automation API provides a standard interface for 3rd party applicat
 </div>
 
 ## Requirements
-The ArcGIS Earth Automation API requires .NET Framework 4.6.1 or later.
+* `ArcGISEarth.AutoAPI.Examples` project and `ArcGISEarth.AutoAPI.Shared` project require .NET Framework 4.6.1 or later.
+* `ArcGISProToArcGISEarth` project requires .NET Framework 4.8 or later.
```
Comment (gis2all) [inline README.md:19]: .Net Core project?

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        {
+            get { return GetBaseUrl(); }
+        }
+
+        #region Automation API Function Tasks
+
+        /// <summary>
+        /// Automation API Functions
+        /// </summary>
+        /// <returns>Automation API response message</returns>
```
Comment (gis2all) [inline src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:46]: Remove this description or all methods add description.

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+            }
+            catch
+            {
+                return DEFAULT_BASEURL + END_POINT;
+            }
+        }
+
+        #endregion
+    }
+}
```
Comment (gis2all) [inline src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:303]: Remove blank lines.

**Review by gis2all — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yong7743) [reply README.md:19]: I will update it.

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs (hunk 2 above)
Comment (yong7743) [reply src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:46]: We should add comments for all the methods in this file. @gis2all @xuew8412

**Comment by yong7743** [comment]
- [ ] @yong7743 needs to update the About information of this repo after merging this PR.

**Review by gis2all — APPROVED** [review]
Looks good.

**Pull request merged**